src/autopas/containers/verletListsCellBased/verletLists/VerletLists.h

```diff
@@ -297,71 +372,267 @@ class VerletLists
   }
 
   /**
-   * update the AoS id maps.
-   * The Id Map is used to map the id of a particle to the actual particle
-   * @return
+   * Loops over cells that are relevant for the inner traversal.
+   * @tparam LoopBody
+   * @param dims
+   * @param parallel define if this is allowed to be parallel
+   * @param loopBody
    */
-  size_t updateIdMapAoS() {
-    size_t i = 0;
+  template <class LoopBody>
+  void innerRelevantTraversal(const std::array<size_t, 3>& dims, bool parallel, LoopBody loopBody) {
```

Comment:
```suggestion
  void innerTraversal(const std::array<size_t, 3>& dims, bool parallel, LoopBody loopBody) {
```
or is there also an irrelevantTraversal?

Reply:
also applies to similar function names